application/jira/v1/config/tasks.json

```diff
@@ -0,0 +1,1181 @@
+          "instillShortDescription": "The status of the issue",
+          "instillUIOrder": 4,
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "type": "string"
+        },
+        "summary": {
+          "title": "Summary",
+          "description": "The summary of the issue",
+          "instillShortDescription": "The summary of the issue",
+          "instillUIOrder": 2,
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "type": "string"
+        },
+        "description": {
+          "title": "Description",
+          "description": "The description of the issue",
+          "instillShortDescription": "The description of the issue",
+          "instillUIOrder": 3,
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "type": "string"
+        },
+        "assignee": {
+          "title": "Assignee",
+          "description": "The assignee of the issue",
+          "instillShortDescription": "The assignee of the issue",
+          "instillUIOrder": 5,
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "type": "string"
+        }
+      },
+      "required": [
+        "project-key",
+        "issue-type",
+        "status"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "Create an issue in Jira",
+      "instillUIOrder": 0,
+      "$ref": "#/$defs/issue",
+      "title": "Output",
+      "type": "object"
+    }
+  },
+  "TASK_UPDATE_ISSUE": {
+    "description": "Update an issue in Jira",
+    "instillShortDescription": "Update an issue in Jira",
+    "input": {
+      "description": "Update an issue in Jira",
+      "instillUIOrder": 0,
+      "instillEditOnNodeFields": [
+        "issue-key",
+        "update"
+      ],
+      "properties": {
+        "issue-key": {
+          "$ref": "#/$defs/issue/properties/key",
+          "title": "Issue Key",
+          "instillUIOrder": 0
+        },
+        "update": {
+          "title": "Update",
+          "description": "The fields to update",
+          "instillUIOrder": 1,
+          "instillFormat": "array:structured/update-field",
+          "instillAcceptFormats": [
+            "array:structured/update-field"
```

Comment:
`structured/update-field`
Do we have this format?

Reply:
I thought we could self-define some format. However, I actually do not quite understand which files should be edited. Any guidance or advice would be greatly appreciated.